content/blog/treetables-1.1.1.md

```diff
@@ -0,0 +1,42 @@
+---
+author: "Alex Hill"
+date: 2019-06-11
+title: jQuery TreeTables 1.1.1
+best: false
+---
+
+We just released a new npm package [TreeTables](https://www.npmjs.com/package/treetables)
+ for displaying tree data when using jQuery
+[DataTables](https://datatables.net/)
+
+DataTables is a powerful and extremely well established plugin for
+displaying tabular data, but has no native support for tree data.
```

Comment:
can you give an example of tree data?

Reply:
Maybe 'nested data' would be more obvious?